lib/reserves.json

```diff
@@ -0,0 +1,269 @@
+    "actions": [
+      {
+        "name": "IC Blinkpack",
+        "activation": "Protocol",
+        "detail": "As a protocol, you may teleport whenever you move for the rest of your turn."
+      }
+    ]
+  },
+  {
+    "id": "reserve_explosive_knuckles",
+    "name": "Explosive Knuckles",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "After hitting with a RAM, you may activate this to deal 6 explosive damage 1/mission."
+  },
+  {
+    "id": "reserve_ultra_high_penetrator_rounds",
+    "name": "Ultra-High Penetrator Rounds",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "After hitting with a ranged attack, you may activate this to declare that the attack’s damage can’t be reduced 1/mission."
+  },
+  {
+    "id": "reserve_run_off_energy_converter",
+    "name": "Run-Off Energy Converter",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "When you use your CORE POWER, you may activate this to generate a BURST 4 aura of lightning. Hostile characters in the area take 4 energy damage 1/mission."
+  },
+  {
+    "id": "reserve_system_backup",
+    "name": "System Backup",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "When forced to perform a save or check you may activate this to automatically succeed 1/mission."
+  },
+  {
+    "id": "reserve_field_marshal_comp_con",
+    "name": "“Field Marshal” Comp/Con",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "As a quick action, up to two allied characters within SENSORS may each immediately perform a single quick action as a reaction 1/mission.",
+    "actions": [
+      {
+        "name": "“Field Marshal” Comp/Con",
+        "activation": "Quick",
+        "detail": "As a quick action, up to two allied characters within SENSORS may each immediately perform a single quick action as a reaction."
+      }
+    ]
+  },
+  {
+    "id": "reserve_retractable_slab_shields",
+    "name": "Retractable Slab Shields",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "As a protocol, you become IMMOBILIZED until the start of your next turn and gain RESISTANCE to all damage for the same duration 1/mission.",
+    "actions": [
+      {
+        "name": "Retractable Slab Shields",
+        "activation": "Protocol",
+        "detail": "As a protocol, you become IMMOBILIZED until the start of your next turn and gain RESISTANCE to all damage for the same duration."
+      }
+    ]
+  },
+  {
+    "id": "reserve_automated_reflex_suite",
+    "name": "Automated Reflex Suite",
+    "type": "Mech",
+    "label": "Resource",
+    "description": "When you are hit by an attack, you may make the attack a miss as a reaction 1/mission.",
+    "actions": [
+      {
+        "name": "Automated Reflex Suite",
+        "activation": "Reaction",
+        "trigger": "When you are hit by an attack.",
+        "detail": "You may make the attack a miss as a reaction."
```

Comment:
Wording is a little weird but gets the point across. I'd probably try this since it's similar to Ace's Barrel Roll or Hornet's Umbral Interdiction:

```json
"trigger": "You are hit by an attack.",
"detail": "The attack automatically misses."
```

Reply:
I was attempting to preserve the working on the table but yes, this seems clearer.